src/components/personal-info.jsx

```diff
@@ -0,0 +1,35 @@
+import React, {useState} from "react";
+
+function PersonalInfo() {
+  const [userInfo, setUserInfo] = useState({
+    name: "",
+    number: "",
+    email: ""
+  })
+
+  const handleChange = (e) => {
+    setUserInfo({...userInfo, [e.target.name]: e.target.value})
+  }
+
```

Comment:
```suggestion

const handleSubmit = async (e) => {
    e.preventDefault();

    try {
      const response = await axios.post("/resume/personal", //update the route after it's implemented in the backend
      ...userInfo);
      console.log(`user info added with ID: ${response.data.id}`);
      navigateTo("mainMenu");
    } catch (error) {
      console.error("Error adding user info:", error);
      alert("Failed to add user info.");
    }
}; 
```

Add a callback function to handle submission event